apps/docs/src/routes/package/+layout.server.ts

```diff
@@ -0,0 +1,33 @@
+import fs from "node:fs/promises";
+import path from "node:path";
+import url from "node:url";
+
+import normalize from "normalize-package-data";
+
+const __filename = url.fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+export async function load() {
+	const root_dir_url = url.pathToFileURL(
+		path.resolve(__dirname, "..", "..", "..", "..", "..", "packages"),
+	);
+	const packages = new Map(
```

Comment:
Hmm, runtime (not build time) file system access like this won’t work on at least Cloudflare Pages/Workers and is questionable even in Node environments like Vercel.

Reply:
Yeah, I need to rethink this part with file system access. I'll put it to a draft for now.

Reply:
I've changed to using Vite feature [`import.meta.glob()`](https://vite.dev/guide/features.html#glob-import). This should do the trick, I hope?